munit/jvm/src/test/scala-2.11/hedgehog/munit/HedgehogSuiteIntegrationTest.scala

```diff
@@ -0,0 +1 @@
+/home/jackcviers/Documents/development/scala-hedgehog/munit/jvm/src/test/scala-3.0/hedgehog/munit/HedgehogSuiteIntegrationTest.scala
```

Comment:
A symlink to the an absolute path.

Reply:
Addressed in https://github.com/hedgehogqa/scala-hedgehog/pull/229/commits/f3b92e35c18d0c3b77846970d1668d8a8c85894b